### PR TITLE
Build 3024 cope with indices, change exchange column operation

### PR DESCRIPTION
**Pull request by mrbray99**
Updated Quote Loader to cope with indices on Yahoo (ticker starts with ^)

### CODE REVIEW

File: src/com/moneydance/modules/features/securityquoteload/GetYahooHistQuote.java [hunk 1]
```diff
@@ -63,10 +64,12 @@
 	private String currencyID="";
 	public GetYahooHistQuote(String tickerp, QuoteListener listenerp, CloseableHttpClient httpClientp,String tickerTypep, String tidp) {
 		super(tickerp, listenerp, httpClientp, tickerTypep,  tidp);
+		String convTicker = ticker.replace("^", "%5E");
```
Comment (sreilly) [inline src/com/moneydance/modules/features/securityquoteload/GetYahooHistQuote.java]: I noticed URLEncoder is imported but not used, even though it seems like you're doing a manual encoding of the ^ sign here.  Did you mean to use URLEncoder.encode() do encode the entire ticker symbol?

File: src/com/moneydance/modules/features/securityquoteload/GetYahooQuote.java [hunk 2]
```diff
@@ -62,10 +63,12 @@
 	private String yahooCurrURL = "https://finance.yahoo.com/quote/";
 	public GetYahooQuote(String tickerp, QuoteListener listenerp, CloseableHttpClient httpClientp,String tickerTypep, String tidp) {
 		super(tickerp, listenerp, httpClientp, tickerTypep,  tidp);
+		String convTicker = ticker.replace("^", "%5E");
```
Comment (sreilly) [inline src/com/moneydance/modules/features/securityquoteload/GetYahooQuote.java]: same thing here

File: src/com/moneydance/modules/features/securityquoteload/loadPricesWindow.java [hunk 3]
```diff
@@ -1452,12 +1452,13 @@ protected String addPriceUrl(String stock, String type){
 		return "&"+queries;
 	}
 	public void testTicker(String url){
-		debugInst.debug("loadPrices", "testTicker", MRBDebug.INFO, "URI "+url);
+		debugInst.debug("loadPricesWindow", "testTicker", MRBDebug.INFO, "URI "+url);
 		URI uri=null;
+		String convUrl = url.replace("^", "%5E");
```
Comment (sreilly) [inline src/com/moneydance/modules/features/securityquoteload/loadPricesWindow.java]: same issue with encoding the ticker symbol

**Review by sreilly — APPROVED** [review]
Looks good, but hopefully my comments are helpful (or at least make sense!)

**Comment by mrbray99** [comment]
Thanks Sean, i removed urlencoder as it fails on a ^ and was causing yahoo quotes to fail. Yahoo uses ^ for indices, eg ^DJI.i should have removed the import.
Regards
Mike

**Pull request merged**